### PR TITLE
adding fixes for the list

### PR DESCRIPTION
**Pull request by dsari-rh**

### CODE REVIEW

**Comment by kumuduh** [comment]
> 

We can remove this section in SCP right ? I don't think it's relevant and it's points to non-STS. I believe we only need market place access right? 

Ensure that your organization's [service control policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) has the minimum set of effective permissions as detailed [here](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.html#rosa-minimum-scp_prerequisites).

**Comment by dsari-rh** [comment]
yeah, i already included the permissions links in the prev section, let me rewrite that part in a bit

File: content/docs/rosa/prereq-list/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  - Connor Wooley
+  - Diana Sari
+---
+
+## Background
+This is a quick checklist of prerequisites needed to spin up a classic [Red Hat OpenShift Service on AWS (ROSA)](https://developers.redhat.com/products/red-hat-openshift-service-on-aws/overview) cluster with [STS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html). Note that this is a high level checklist and your implementation may vary. 
+
+Before running the installation process, make sure that you deploy this from a machine that has access to:
+- The API services for the cloud to which you provision.
+- Access to `api.openshift.com` and `sso.openshift.com`.
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:16]: sorry this is actually `sso.redhat.com`

File: content/docs/rosa/prereq-list/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+First, let's discuss about the accounts and CLIs you would need to install to deploy the cluster.
+
+### AWS account:
+  - You would need the following details:
+      - AWS IAM User
+      - AWS Access Key ID
+      - AWS Secret Access Key
+  - Ensure that you have the right permissions as detailed [here](https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol.html) and [here](https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html)
+  - Please also refer [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#rosa-account_rosa-sts-aws-prereqs) for more details. 
+### AWS CLI (`aws`):
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:32]: markdown elements like tables or code  blocks or lists should have a empty line above and below them.

File: content/docs/rosa/prereq-list/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  - Install from [here](https://aws.amazon.com/cli/) if you have not already.
+  - Configure the CLI:
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:34]: These should probably be an ordered list as they're steps we want the user to take.

File: content/docs/rosa/prereq-list/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+- When you create a ROSA cluster using AWS STS, an associated AWS OpenID Connect (OIDC) identity provider is created as well. 
+    - This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. 
+    - Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.
+
+
+## Networking Prerequisites
+Next, let's talk about the prerequisites needed from networking standpoint.
+
+### Firewall
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:79]: do we need proxy considerations as well as Firewall ?

File: content/docs/rosa/prereq-list/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    - The subnet CIDR blocks must belong to the machine CIDR that you specify.
+- The VPC must not use the `kubernetes.io/cluster/.*: owned`, `Name`, and `openshift.io/cluster` tags.
+- Set both `enableDnsHostnames` and `enableDnsSupport` to `true`.
+    - That way, the cluster can use the Route 53 zones that are attached to the VPC to resolve cluster’s internal DNS records.
+- Verify route tables by running `aws ec2 describe-route-tables --filters "Name=vpc-id,Values=<vpc-id>"`. 
+    - Ensure that the cluster can egress either via NAT gateway in public subnet or via transit gateway.
+    - And ensure whatever UDR you would like to follow is set up.
+    
+
+## ROSA Prerequisites without STS
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:101]: We should tell them that nonSTS clusters are deprecated and soon new clusters will only be STS.  Don't encourage them to go do mint mode.

File: content/docs/rosa/prereq-list/index.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+      - Otherwise, the upstream DNS will need to forward the cluster scope to this VPC so the cluster can resolve internal IPs/services.
+
+## PrivateLink Prerequisites
+If you would like to deploy a PrivateLink cluster, then be sure to deploy the cluster in the pre-existing VPC (BYO VPC) and please refer [here](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-vpc.html) for more details and below in high level:
+
+- Create a public and private subnet for each AZ that your cluster uses.
+    - Alternatively, implement transit gateway for internet/egress with appropriate routes.
+- The VPC's CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address for cluster machines. 
+    - The subnet CIDR blocks must belong to the machine CIDR that you specify.
+- The VPC must not use the `kubernetes.io/cluster/.*: owned`, `Name`, and `openshift.io/cluster` tags.
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:93]: the VPC must not have .... tags, as the ROSA installer will need to tag the VPC with the cluster identifier when its installed.

File: content/docs/rosa/prereq-list/index.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+Next, let's talk about the prerequisites needed from networking standpoint.
+
+### Firewall
+  - Configure your firewall to allow access to the domains and ports listed [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs)
+### Custom DNS
+  - If you want to use custom DNS, then ROSA installer must be able to use VPC DNS with default DHCP options so it can resolve hosts locally. 
+      - To do so, run `aws ec2 describe-dhcp-options` and see if the VPC is using VPC Resolver.
+      - Otherwise, the upstream DNS will need to forward the cluster scope to this VPC so the cluster can resolve internal IPs/services.
+
+## PrivateLink Prerequisites
```
Comment (paulczar) [inline content/docs/rosa/prereq-list/index.md:86]: non private-link VPCs also support BYO-VPC.  we should call out those requirements separately.

File: content/docs/rosa/prereq-list/index.md (hunk 1 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:16]: ok

File: content/docs/rosa/prereq-list/index.md (hunk 2 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:32]: theres no tables nor code blocks in there, but i'll give an extra line if that helps

File: content/docs/rosa/prereq-list/index.md (hunk 3 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:34]: the configure the cli part you mean? ok on it

File: content/docs/rosa/prereq-list/index.md (hunk 4 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:79]: i'll add this on the privatelink section

File: content/docs/rosa/prereq-list/index.md (hunk 5 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:101]: i'll just chop this part off

File: content/docs/rosa/prereq-list/index.md (hunk 6 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:93]: ok

File: content/docs/rosa/prereq-list/index.md (hunk 7 above)
Comment (dsari-rh) [reply content/docs/rosa/prereq-list/index.md:86]: i'll add a note at the end

File: content/docs/rosa/prereq-list/index.md (hunk 2 above)
Comment (paulczar) [reply content/docs/rosa/prereq-list/index.md:33]: yeah sorry I was just giving the examples ... headings should also have a blank line before or after.  basically any kind of markdown element that's not specifically inline.  This comment was meant to apply to the whole doc, just chose this line to pick on.

**Review by cwooley-rh — APPROVED** [review]

**Pull request merged**